deploy/lib/server_config.rb

```diff
@@ -2058,6 +2058,37 @@ def deploy_rest
       deploy_rest_config()
       deploy_ext()
       deploy_transform()
+
+      if !test &&
```

Comment:
Whats the point of `test` ? Would it only be true if called by app_specific.rb ?